torchvision/csrc/ops/quantized/cpu/qroi_align_kernel.cpp

```diff
@@ -0,0 +1,201 @@
+#include <ATen/ATen.h>
+#include <ATen/native/quantized/affine_quantizer.h>
+#include <torch/library.h>
+
+#include "../../roi_align.h"
+
+namespace vision {
+namespace ops {
+
+namespace {
+
+template <typename T>
+void qroi_align_forward_kernel_impl(
```

Comment:
reviewing this file might be easier by doing a `vimdiff` or similar with `ops/cpu/roi_align_kernel.cpp`